src/ethz_snow/config/snowConfig_default.yaml

```diff
@@ -1,36 +1,87 @@
+# temperature resolution within vial (homogeneous, spatial_1D, spatial_2D)
+temperature: spatial_2D
+
+# freezing configuration to be simulated (shelf, VISF, jacket)
+configuration: jacket
+
+# all vial-related parameters
 vial:
+  # define the geometry
   geometry:
-    # base shape of vial (currently only cube is accepted)
-    shape: cube
-    # length [m]
+    # base shape of vial (in the spatial version only cylinder is accepted)
+    shape: cylinder
+    # diameter of the vial for cylindrical geometry [m]
+    diameter: 0.01
+    # height of the filled product for cylindrical geometry [m]
+    height: 0.01
+    # length of the vial when shape is cube [m]
     length: 0.01
-    # width [m]
+    # width of the vial when shape is cube [m]
     width: 0.01
-    # height [m]
-    height: 0.01
+
+# parameters used in VISF simulation
+VISF:
+  # vacuum pressure [Pa]
+  p_vac: 100
+  # evaporation coefficient [-]
+  kappa: 0.01
+  # latent heat of vaporization for water [J/kg]
+  Dh_evaporation: 2500.9e3
+  # mass of water molecule [kg]
+  m_water: 2.99e-26
+  # time for vacuum start [h]
+  t_vac_start: 0.75
+  # duration of the vacuum [h]
+  t_vac_duration: 0.1
+
+# parameters for jacket-ramped freezing
+jacket:
+  # air gap between the wall and the vial [m]
+  air_gap: 0.001
+  # heat conductivity of air [W/mK]
+  lambda_air: 0.025
+
+# all parameters realted to water
 water:
-  # J/[Kkg] heat capacity liquid water
+  # specific heat capacity of liquid water [J/kgK]
   cp_w: 4187
-  # J/[Kkg] heat capacity ice 0°C
+  # specific heat capacity of ice [J/kgK]
   cp_i: 2108
-  # J/kg heat of fusion water
+  # heat conductivity of liquid water [W/mK]
+  lambda_w: 0.598
+  # heat conductivity of ice [W/mK]
+  lambda_i: 2.25
+  # latent heat of fusion of water [J/kg]
   Dh: 333550
+
+# all solution parameters
 solution:
-  # kg/m^3 density of water / assumed constant for all phases
+  # solute mass fraction [-]
+  solid_fraction: 0.25
```

Comment:
how does this change/increase affect snow and snowfall?

Reply:
Sorry, default should stay 0.05. I changed it back.